waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -538,7 +539,7 @@
                         :k8s/node-name "node-0.k8s.com"
                         :k8s/pod-phase "Pending"
                         :log-directory "/home/myself/r0"
-                        :port 8080
+                        :port (generate-port-with-offset "test-app-1234" 8080 :backward 0)
```

Comment:
Let's keep the port number hard-coded in the expected value.

Reply:
Done